### ISSUE TITLE
Python regression tests are failing

### ISSUE THREAD

**Issue by joequant** [AIDASoft/podio#151]
- OS version: e.g. SLC6 Mageia
 - Compiler version: e.g GCC 6.2 10.2
 - PODIO version: tag or commit ID, or GitHub branch 9c86507fcf327058edc1d9e1d6d3e6c2ecc9bea9

- Reproduced by: exact steps to reproduce the problem: checkout, setup environment (Geant, ROOT versions, iLCSoft release), cmake, build, run...

Using ROOT 6.22.0

Build and run make tests

 - Goal: A short description of what you are trying to achieve

The python regression tests are failing

The first issue is that the variable LD_LIBRARY_PATH is not set so that python cannot find the .so files that are build in the compile.

Once I set the environment variables, I'm getting this error....

```
======================================================================
ERROR: test_chain (test_EventStore.EventStoreTestCase)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/joe/local2/bench/git/podio/python/test_EventStore.py", line 91, in test_chain
    numbers.append(evinfo[0].Number())
TypeError: 'CollectionBase' object is not subscriptable

======================================================================
ERROR: test_collections (test_EventStore.EventStoreTestCase)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/joe/local2/bench/git/podio/python/test_EventStore.py", line 33, in test_collections
    self.assertTrue(len(evinfo) > 0)
ReferenceError: attempt to access a null-pointer

======================================================================
ERROR: test_hash (test_EventStore.EventStoreTestCase)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/joe/local2/bench/git/podio/python/test_EventStore.py", line 66, in test_hash
    for cluster in clusters:
TypeError: 'CollectionBase' object is not iterable

======================================================================
ERROR: test_one_to_many (test_EventStore.EventStoreTestCase)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/joe/local2/bench/git/podio/python/test_EventStore.py", line 49, in test_one_to_many
    for cluster in clusters:
TypeError: 'CollectionBase' object is not iterable
```

**Comment by tmadlener** [comment]
Hi @joequant,

Are the regression tests running on podio master? In principle this issue should already be addressed by #141.

**Comment by joequant** [comment]
Okay.  I think my tree got corrupted.  I deleted everything and then checked everything out with a clean build, and it works.
